proto/rpc/user/user.proto

```diff
@@ -75,8 +75,18 @@ message UserForgotPasswordResponse { string recoveryLink = 1; }
 
 message UserDeleteApiKeysResponse {}
 message CliConfigRequest {}
+message AuditWebhook {}
```

Comment:
create two messages to follow buf standards, UserLoginAuditRequest and UserLoginAuditResponse